be/src/exprs/bitmap_function.cpp

```diff
@@ -465,6 +465,43 @@ StringVal BitmapFunctions::bitmap_from_string(FunctionContext* ctx, const String
     return result;
 }
 
+BooleanVal BitmapFunctions::bitmap_contains(FunctionContext *ctx, const StringVal &src, const IntVal &input) {
```

Comment:
Because we don't have unsigned int, but the bit value can exceede the max int. Better use BigIntValue as bit index